pkgs/development/libraries/webkitgtk/default.nix

```
@@ -189,9 +187,11 @@ stdenv.mkDerivation rec {
     "-DUSE_APPLE_ICU=OFF"
     "-DUSE_OPENGL_OR_ES=OFF"
     "-DUSE_SYSTEM_MALLOC=ON"
-  ] ++ lib.optionals (!stdenv.isLinux) [
-    "-DUSE_SYSTEMD=OFF"
-  ] ++ lib.optional (stdenv.isLinux && enableGLES) "-DENABLE_GLES2=ON";
+  ]
+  ++ lib.optional (!stdenv.isLinux) "-DUSE_SYSTEMD=OFF"
+  ++ lib.optional (stdenv.isLinux && enableGLES) "-DENABLE_GLES2=ON"
+  ++ lib.optional (libsecret == null) "-DUSE_LIBSECRET=OFF"
+  ;
```

Could you please use the GNOME style (no `optional`, one value per line) for GNOME packages?

```suggestion
  ] ++ lib.optionals (!stdenv.isLinux) [
    "-DUSE_SYSTEMD=OFF"
  ] ++ lib.optionals (stdenv.isLinux && enableGLES) [
    "-DENABLE_GLES2=ON"
  ] ++ lib.optionals (libsecret == null) [
    "-DUSE_LIBSECRET=OFF"
  ];
```